docs/kyma/04-02-local-installation.md

```diff
@@ -23,84 +24,99 @@ Virtualization:
+  
+  4. Export the release version as an environment variable:
+     ```bash
+     export KYMA_RELEASE={KYMA_RELEASE_VERSION}
+     ```
+  5. Install Kyma release `$KYMA_RELEASE` in minikube:
+     ```bash
+     kyma install -r $KYMA_RELEASE
+     ```
+     
+     Wait until installation process completed
+     
+  6. After the installation is completed, you can access the Console UI. Go to [this](https://console.kyma.local) address and select **Login with Email**. Use the **admin@kyma.cx** email address and the password printed in the terminal after installation finished.
+  
+  7. At this point, Kyma is ready for you to explore. See what you can achieve using the Console UI or check out one of the [available examples](https://github.com/kyma-project/examples).
+   
+  </details>
+  <details>
+  <summary>
+  From sources
+  </summary>
+    
+  1. Open a terminal window and navigate to a space in which you want to store local Kyma sources.
+    
+  2. Clone the Kyma repository to your machine using either HTTPS or SHH. Run this command to clone the repository.
+     
+     <div tabs>
+       <details>
+       <summary>
+       HTTPS
+       </summary>
+     
+        ```bash
+        git clone https://github.com/kyma-project/kyma.git
+        ```
+       </details>
+       <details>
+       <summary>
+       SSH
+       </summary>
+     
+       ```bash
+       git clone git@github.com:kyma-project/kyma.git
+       ```
+       </details>
+     </div>
+ 
+  3. Kyma comes with a local wildcard self-signed `server.crt` certificate. Add this certificate to your OS trusted certificates to access the Console UI. On MacOS, run:
+     ```bash
+     sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain certs/workspace/raw/server.crt
+     ```
+     >**NOTE:** Mozilla Firefox uses its own certificate keychain. If you want to access the Console UI though Firefox, add the Kyma wildcard certificate to the certificate keychain of the browser. To access the Application Connector and connect an external solution to the local deployment of Kyma, you must add the certificate to the trusted certificate storage of your programming environment. Read [this](/components/application-connector#details-access-the-application-connector-on-a-local-kyma-deployment) document to learn more.
+
+  4. Provision a kubernetes cluster on minikube ready to be installed with Kyma run this command:
+     ```bash
+     kyma provision minikube
+     ```
+     > **NOTE:** Kyma CLI command above uses default minikube vm driver installed for your OS, for list of default vm driver for supported OS and other available vm drivers you can read from [here](http://github.com/kyma-project/cli)
+  5. Install Kyma with following command with password for the `admin@kyma.cx` user:
+     
+     ```bash
+     kyma install --local --src-path {YOUR_KYMA_SOURCE_PATH} --password {YOUR_PASSWORD}
```

Comment:
src-path and password are optional, I think we should document here the minimum for the default flow and just link the cli docu for more options.

IN the specific case let's remove the password and in a note mention that src-path is required in case you have not checked out in the GO_PTAH

Reply:
We make assumption most of developer will use git clone without specifying clone directory to the GO_PATH